dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/header/HeaderExchangeClient.java

```diff
@@ -210,6 +215,38 @@ private long calculateLeastDuration(int time) {
         }
     }
 
+    private boolean shouldHeartbeat(URL url) {

```

Comment:
Same with shouldStartCloseTimer.
Seems like these two methods can be merged into one method which named like `isNettyTransport` or sth...